test/integration/framework/shoot_app.go

```diff
@@ -58,18 +55,18 @@ const (
 // NewGardenTestOperation initializes a new test operation from created shoot Objects that can be used to issue commands against seeds and shoots
 func NewGardenTestOperation(ctx context.Context, k8sGardenClient kubernetes.Interface, logger logrus.FieldLogger, shoot *v1beta1.Shoot) (*GardenerTestOperation, error) {
 	if err := k8sGardenClient.Client().Get(ctx, client.ObjectKey{Namespace: shoot.Namespace, Name: shoot.Name}, shoot); err != nil {
-		return nil, err
+		return nil, errors.Wrapf(err, "could not get Shoot in Garden cluster:")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "could not get Shoot in Garden cluster")
```

You don't need Wrapf, you could use Wrap. You don't need the extra ":", the wrapping will add it.